server/routes/user.js

```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+import Users from '../controllers/Users';
+
+const route = express.Router();
+
+route.post('/create', Users.signUp);
```

Comment:
pls change to: `route.post('/users/create', Users.create)`
This means you have to change the method name.
Make this api change also reflect on the README.md

Reply:
Noted